Tests/check_j2k_overflow.py

```diff
@@ -1,13 +1,13 @@
 from __future__ import annotations
 
-from pathlib import PosixPath
+from pathlib import Path
 
 import pytest
 
 from PIL import Image
 
 
-def test_j2k_overflow(tmp_path: PosixPath) -> None:
+def test_j2k_overflow(tmp_path: Path) -> None:
```

Comment:
I think we should we use `os.PathLike` instead of `pathlib.Path`? (here and below)

https://docs.python.org/3/library/os.html#os.PathLike

Reply:
https://docs.pytest.org/en/6.2.x/tmpdir.html
> tmp_path is a pathlib.Path object.

?